activerecord-userstamp.gemspec

```diff
@@ -16,17 +16,25 @@ Gem::Specification.new do |s|
 
   s.files         = `git ls-files`.split("\n")
   s.test_files    = `git ls-files -- {test,spec,features}/*`.split("\n")
-  s.executables   = `git ls-files -- bin/*`.split("\n").map{ |f| File.basename(f) }
+  s.executables   = `git ls-files -- bin/*`.split("\n").map { |f| File.basename(f) }
   s.require_paths = ['lib']
 
-  s.add_dependency 'activesupport', '~> 4.2'
-  s.add_dependency 'activerecord', '~> 4.2'
+  rails_version = ENV['RAILS_VERSION'] || 'default'
```

Comment:
For this check, it might be good to also check for `ENV['CI'] == 'true'`, to only affect this when running on Travis.